packages/ui-extensions/docs/surfaces/customer-account/staticPages/configuration.doc.ts

```diff
@@ -218,9 +218,11 @@ The following section describes use cases for requesting network access, alterna
         {
           title: 'Alternatives to network access',
           sectionContent: `
-Instead of fetching data with an external network call, consider retrieving the data from a metafield. Your app may be able to use the [Admin API](/docs/api/admin) to write [metafields](/api/admin-graphql/latest/objects/metafield) on the shop, product, or customer.
+Instead of fetching data with an external network call, consider retrieving the data from a metafield. Your app can use the [Customer Account API](/docs/api/customer) to write to [metafields](/docs/api/customer/latest/objects/Metafield) on the customer, order, company, or company location.
 
-Retrieving data from [metafields](/docs/api/customer-account-ui-extensions/apis/standardapi#properties-propertydetail-appmetafields) in new customer accounts is faster since it won't introduce an external network call. This allows you to rely on Shopify for the uptime, scaling, and durability of the data storage.
+Your app may be able to use the [Admin API](/docs/api/admin) to write to [metafields](/api/admin-graphql/latest/objects/metafield) on the shop or product.
```

Comment:
Hmm...good question. Again, this is what's in the existing content.

First reason that comes to mind is that it's because an extension-only app would not be able to do this since you'd need a backend to proxy a request to the Admin API

Maybe something like: `An app with a back-end can use the Admin API to...`

But this kind of negates the following statement about why metafields should be used since we're making an extra network request.